RealmSwift.xcodeproj/project.pbxproj

```diff
@@ -0,0 +1,1351 @@
+			files = (
+				020F63F81A8037DC007CB52E /* SwiftUnicodeTests.swift in Sources */,
+				C0E04FA51AB235D900DCC8C0 /* ObjectSchemaInitializationTests.swift in Sources */,
+				C0E04FA61AB235E000DCC8C0 /* SortDescriptorTests.swift in Sources */,
+				C0E04F9F1AB235CE00DCC8C0 /* ObjectAccessorTests.swift in Sources */,
+				020F63FB1A8037DC007CB52E /* TestCase.swift in Sources */,
+				C0E04FA21AB235D400DCC8C0 /* ObjectCreationTests.swift in Sources */,
+				020F63F51A8037DC007CB52E /* SwiftTestObjects.swift in Sources */,
+				C081AEB41AAF8FFB00BA3E31 /* PropertyTests.swift in Sources */,
+				020F63E31A8037DC007CB52E /* ResultsTests.swift in Sources */,
+				C0240C2D1A84090000858BFA /* RealmTests.swift in Sources */,
+				C0027B0D1AAF736700F013A2 /* ObjectSchemaTests.swift in Sources */,
+				020F63EF1A8037DC007CB52E /* SwiftPropertyTypeTest.swift in Sources */,
+				C0027B091AAF6FBC00F013A2 /* SchemaTests.swift in Sources */,
+				0253CDAC1AAE667000AF9E6C /* TestUtils.mm in Sources */,
+				020F63E91A8037DC007CB52E /* SwiftLinkTests.swift in Sources */,
+				020F63E01A8037DC007CB52E /* ListTests.swift in Sources */,
+				C0FE465E1A9BE7A6006E0539 /* ObjectTests.swift in Sources */,
+				02616BD71A817FC8009919CB /* MigrationTests.swift in Sources */,
+			);
+			runOnlyForDeploymentPostprocessing = 0;
+		};
+		C0E04FA71AB2366200DCC8C0 /* Sources */ = {
+			isa = PBXSourcesBuildPhase;
+			buildActionMask = 2147483647;
+			files = (
+				C0E04FD11AB2367300DCC8C0 /* main.m in Sources */,
+			);
+			runOnlyForDeploymentPostprocessing = 0;
+		};
+/* End PBXSourcesBuildPhase section */
+
+/* Begin PBXTargetDependency section */
+		0201B2991A54C7BD004CFB6F /* PBXTargetDependency */ = {
+			isa = PBXTargetDependency;
+			target = 0201B2661A54C68C004CFB6F /* RealmSwift OSX */;
+			targetProxy = 0201B2981A54C7BD004CFB6F /* PBXContainerItemProxy */;
+		};
+		026B0F011A780521005E26C8 /* PBXTargetDependency */ = {
+			isa = PBXTargetDependency;
+			target = 026B0EFB1A780414005E26C8 /* Realm iOS */;
+			targetProxy = 026B0F001A780521005E26C8 /* PBXContainerItemProxy */;
+		};
+		02B8228F1A532125007F28A5 /* PBXTargetDependency */ = {
+			isa = PBXTargetDependency;
+			target = 023B19A71A4234380067FB81 /* RealmSwift iOS */;
+			targetProxy = 02B8228E1A532125007F28A5 /* PBXContainerItemProxy */;
+		};
+		295C17A91A7C5627000E5F54 /* PBXTargetDependency */ = {
+			isa = PBXTargetDependency;
+			target = 295C17A31A7C55DA000E5F54 /* Realm OSX */;
+			targetProxy = 295C17A81A7C5627000E5F54 /* PBXContainerItemProxy */;
+		};
+		C0E04FD31AB2367A00DCC8C0 /* PBXTargetDependency */ = {
+			isa = PBXTargetDependency;
+			target = C0E04FAA1AB2366200DCC8C0 /* TestHost */;
+			targetProxy = C0E04FD21AB2367A00DCC8C0 /* PBXContainerItemProxy */;
+		};
+/* End PBXTargetDependency section */
+
+/* Begin XCBuildConfiguration section */
+		0201B2791A54C68C004CFB6F /* Debug */ = {
+			isa = XCBuildConfiguration;
+			buildSettings = {
+				CLANG_CXX_LANGUAGE_STANDARD = "gnu++0x";
```

Comment:
Shouldn't matter unless we add C++ files to this project, but this should be c++14.